unit-tests/functional/alt-IR/test-sanity.cpp

```diff
@@ -0,0 +1,38 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "../../tests.h"
+
+using namespace rs2;
+
+TEST_CASE( "AltIR", "[l500][live]" )
+{
+   
+    auto devices = find_device_by_product_line_or_exit( RS2_PRODUCT_LINE_L500 );
```

Comment:
I think we should make it clear by having `devices` in the name rather than `device`

Reply:
done